tests/test_counting_hash.py

```diff
@@ -442,6 +469,114 @@ def test_2_kadian():
     assert x == 1, x
 
 
+def test_get_kmer_counts_too_short():
+    hi = khmer.new_counting_hash(6, 1e6, 2)
+
+    hi.consume("AAAAAA")
+    counts = hi.get_kmer_counts("A")
+    assert len(counts) == 0
+
+
+def test_get_kmer_counts_too_short():
+    hi = khmer.new_counting_hash(6, 1e6, 2)
+
+    hi.consume("AAAAAA")
+    counts = hi.get_kmer_counts("A")
+    assert len(counts) == 0
+
+
+def test_get_kmers_too_short():
+    hi = khmer.new_counting_hash(6, 1e6, 2)
+
+    hi.consume("AAAAAA")
+    kmers = hi.get_kmers("A")
+    assert len(kmers) == 0
+
+
+def test_get_kmer_counts():
+    hi = khmer.new_counting_hash(6, 1e6, 2)
+
+    hi.consume("AAAAAA")
+    counts = hi.get_kmer_counts("AAAAAA")
+    print counts
+    assert len(counts) == 1
+    assert counts[0] == 1
+
+    hi.consume("AAAAAA")
+    counts = hi.get_kmer_counts("AAAAAA")
+    print counts
+    assert len(counts) == 1
+    assert counts[0] == 2
+
+    hi.consume("AAAAAT")
+    counts = hi.get_kmer_counts("AAAAAAT")
+    print counts
+    assert len(counts) == 2
+    assert counts[0] == 2
+    assert counts[1] == 1
+
+    hi.consume("AAAAAT")
+    counts = hi.get_kmer_counts("AAAAAAT")
+    print counts
+    assert len(counts) == 2
+    assert counts[0] == 2
+    assert counts[1] == 2
+
+    hi.consume("AAAAAT")
+    counts = hi.get_kmer_counts("AAAAAAT")
+    print counts
+    assert len(counts) == 2
+    assert counts[0] == 2
+    assert counts[1] == 3
+
+
+def test_get_kmer_hashes():
+    hi = khmer.new_counting_hash(6, 1e6, 2)
+
+    hi.consume("AAAAAA")
+    hashes = hi.get_kmer_hashes("AAAAAA")
+    print hashes
+    assert len(hashes) == 1
+    assert hi.get(hashes[0]) == 1
+
+    hi.consume("AAAAAA")
+    hashes = hi.get_kmer_hashes("AAAAAA")
+    print hashes
+    assert len(hashes) == 1
+    assert hi.get(hashes[0]) == 2
+
+    hi.consume("AAAAAT")
+    hashes = hi.get_kmer_hashes("AAAAAAT")
+    print hashes
+    assert len(hashes) == 2
+    assert hi.get(hashes[0]) == 2
+    assert hi.get(hashes[1]) == 1
+
+    hi.consume("AAAAAT")
+    hashes = hi.get_kmer_hashes("AAAAAAT")
+    print hashes
+    assert len(hashes) == 2
+    assert hi.get(hashes[0]) == 2
+    assert hi.get(hashes[1]) == 2
+
+    hi.consume("AAAAAT")
+    hashes = hi.get_kmer_hashes("AAAAAAT")
+    print hashes
+    assert len(hashes) == 2
+    assert hi.get(hashes[0]) == 2
+    assert hi.get(hashes[1]) == 3
+
+
+def test_get_kmers():
+    hi = khmer.new_counting_hash(6, 1e6, 2)
```

Comment:
> > +def test_get_kmers():
> > +    hi = khmer.new_counting_hash(6, 1e6, 2)
> 
> too large and not needed

While I appreciate that it's unnecessary, this is only 2 MB ;).  I'll fix it
but -0 on this kind of thing in the future.